.github/workflows/integration-tests-release.yaml

```diff
@@ -0,0 +1,219 @@
+---
+name: Integration Tests and Release
+
+on: 
+  pull_request:
+    paths:
+      - 'pkg/**'
+      - 'cmd/**'
+      - 'charts/**'
+      - 'tests/integration/**'
+      - '.github/workflows/**'
+  push:
+    branches:
+      - 'main'
+      - '**-run-ci'
+      - '*.*'
+    paths:
+      - 'pkg/**'
+      - 'cmd/**'
+      - 'charts/**'
+      - 'tests/integration/**'
+      - '.github/workflows/**'
+  release:
+    types:
+      - published
+
+jobs:
+  integration-tests:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        cassandra_version: ["3.11.10", "4.0.0"]
+        scenario: ["TestMedusaDeploymentScenario/\"S3\"", "TestMedusaDeploymentScenario/\"Minio\"", "TestMedusaDeploymentScenario/\"local\"", "TestReaperDeploymentScenario", "TestMonitoringDeploymentScenario", "TestStargateDeploymentScenario"]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-k8ssandra-buildx
+          restore-keys: |
+            ${{ runner.os }}-k8ssandra-buildx
+      - name: Set up Go 1.15
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.15
+      - name: Install Dependencies
+        run: |
+          set -x
+          sudo apt-get update || echo "something went slightly wrong with apt-get update"
+
+          # install kubectl
+          curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+          sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+
+          # install kind
```

Comment:
Other repos use kind script + helm script to install those dependencies.

Reply:
Which scripts are these? Helm is installed using `./scripts/install-helm.sh` but I don't see an equivalent for kind.

Reply:
```yaml
      - name: Install Helm
        uses: azure/setup-helm@v1
        with:
          version: v3.5.3
      - name: Create Kind Cluster
        uses: helm/kind-action@v1.1.0
        with:
          version: v0.10.0
          node_image: kindest/node:v1.20.2
          cluster_name: kind
          config: tests/testdata/kind/kind_config_6_workers.yaml
```

Reply:
I've made the changes to use your recommended actions instead of using bash to install helm and create the kind clusters.